docker-compose.yml

```diff
@@ -20,7 +19,7 @@ services:
   ui:
     container_name: ui
     build: ./ui
-    image: railsgirlscph/mentormenteeplatform-ui
+    image: railsgirlscph/mentormenteeplatform-ui:latest
```

Comment:
What this does: This uses a **Docker tag** called `latest` to point out _which_ image to run.